### PR TITLE
Nobu

### PR DESCRIPTION
**Pull request by nobuyuki-tono**
I finished AboutUs page

### CODE REVIEW

File: src/GrobalSass/mixin.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@mixin break($size, $isMax: false) {
```
Comment (Vlad-Vekslyer) [inline src/GrobalSass/mixin.scss]: Please provide more detailed comments about how break works

File: src/GrobalSass/mixin.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      md: 768px,
+    // Medium large screen / desktop narrow
+      mdlg: 968px,
+    // Large screen / desktop
+      lg: 1080px,
+    // Extra large screen / wide desktop
+      xl: 1370px,
+    xxl: 1600px
+  );
+
```
Comment (Vlad-Vekslyer) [inline src/GrobalSass/mixin.scss]: Comments before the if statements would be helpful

Comment (Vlad-Vekslyer) [inline src/GrobalSass/mixin.scss:44]: Comments to explain how the if statements work would be helpful

**Review by Vlad-Vekslyer — CHANGES_REQUESTED** [review]

File: src/GrobalSass/mixin.scss (hunk 2 above)
Comment (nobuyuki-tono) [reply src/GrobalSass/mixin.scss:44]: I added comment

**Review by Vlad-Vekslyer — APPROVED** [review]

**Pull request merged**